src/Utils/schedules.jl

```diff
@@ -149,6 +149,37 @@ function specified_times_str(st)
     return string("[", str_elems..., "]")
 end
 
+#####
+##### ConsecutiveIterations
+#####
+
+mutable struct ConsecutiveIterations{S} <: AbstractSchedule
+    parent :: S
+    previous_parent_actuation_iteration :: Int
+end
+
+"""
+    ConsecutiveIterations(parent_schedule)
+
+Return a `schedule::ConsecutiveIterations` that actuates both when `parent_schedule`
+actuates, and at iterations immediately following the actuation of `parent_schedule`.
+"""
```

Comment:
```suggestion
actuates, and at iterations immediately following the actuation of `parent_schedule`.
This can be used, for example, when one wants to use output to numerically reconstruct the
time-derivative of a quantity.
"""
```